### DISCUSSION TITLE
How to achieve something similar to express.router?

### DISCUSSION THREAD

**Discussion by sqllyw** [elysiajs/elysia#609] [Q&A]
https://expressjs.com/en/guide/routing.html

Hi, 

here are some codes from the above link, how can we do similar things in Elysia?
````
const express = require('express')
const router = express.Router()

// middleware that is specific to this router
const timeLog = (req, res, next) => {
  console.log('Time: ', Date.now())
  next()
}
router.use(timeLog)

// define the home page route
router.get('/', (req, res) => {
  res.send('Birds home page')
})
// define the about route
router.get('/about', (req, res) => {
  res.send('About birds')
})

module.exports = router
````
````
const birds = require('./birds')

// ...

app.use('/birds', birds)
````
basically, the routes are defined in a separate file, I was looking at Elysia's group, but don't know how to define related routes in a file,
Thanks

**Comment by kravetsone** [comment]
![IMG_20240414_125209_044.jpg](https://github.com/elysiajs/elysia/assets/57632712/68865934-54a9-47c3-8f15-256a96fb9df8)

**Comment by sqllyw** [comment]
got it, thanks
